CONTRIBUTING.md

```diff
@@ -88,6 +88,15 @@ Then to check if Hugepages is enabled:
 python3 infra/machine_setup/scripts/setup_hugepages.py check
 ```
 
+### Software dependencies for codebase contributions
+1. Install dependencies
+
+```sh
+sudo apt install pandoc libtbb-dev libcapstone-dev pkg-config
```

Comment:
I can't imagine why these things would be needed to contribute.

Contributing only requires pre-commit as far as I'm aware.

Reply:
![Uploading image.png…]()

Reply:
<img width="442" alt="image" src="https://github.com/user-attachments/assets/f944fdce-498d-4418-9cf9-463853e0e3ee" />

Reply:
@tt-rkim can you weight in on this? that step is listed as an optional step for contributors in current version of INSTALLING.md.

Reply:
This is for profiling, which developers often use.

Should we make an additional section for source users who want to use the profiler?